From eaabd91ea030d52932d2d53d3da17cd0bc03a118 Mon Sep 17 00:00:00 2001
From: Adam Lackorzynski <adam@l4re.org>
Date: Fri, 31 Oct 2025 11:51:39 +0100
Subject: [PATCH] configure_sigs.sh: Use less ambiguous identifier

'number' might be used in the included header file(s) which then defeats
the grep for the number. Thus use the less ambiguous identifier to avoid
those type of clashes.

Signed-off-by: Adam Lackorzynski <adam@l4re.org>
---
 cnf/configure_sigs.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cnf/configure_sigs.sh b/cnf/configure_sigs.sh
index d0d1b46..7c07846 100644
--- a/cnf/configure_sigs.sh
+++ b/cnf/configure_sigs.sh
@@ -18,15 +18,15 @@ for sig in HUP INT QUIT ILL TRAP ABRT BUS FPE KILL USR1\
 
 	# OH SHI--
 	try_add "#if SIG$sig == 0"
-	try_add "number 0"
+	try_add "perl_cross_configure_sig_number 0"
 	for num in `seq 1 100`; do
 		try_add "#elif SIG$sig == $num"
-		try_add "number $num"
+		try_add "perl_cross_configure_sig_number $num"
 	done
 	try_add "#endif"
 
 	if try_preproc; then
-		num=`grep 'number ' try.out | sed -e 's/[^0-9]//g'`
+		num=`grep 'perl_cross_configure_sig_number ' try.out | sed -e 's/[^0-9]//g'`
 		if [ -n "$num" -a "$num" != 0 ]; then
 			msg "   got SIG$sig = $num" >&2
 			signals="$signals $sig"
-- 
2.51.0

